run.go

```diff
@@ -53,8 +53,6 @@ const (
 	RunApplying           RunStatus = "applying"
 	RunCanceled           RunStatus = "canceled"
 	RunConfirmed          RunStatus = "confirmed"
-	RunCostEstimated      RunStatus = "cost_estimated"
-	RunCostEstimating     RunStatus = "cost_estimating"
```

Comment:
I think we should keep these states as in some cases they can be returned right?

Reply:
I think that, for now, these are not possible to see in production. If they end up changing, we'll have to remove/replace them anyway, which might be easier to PR as a group.

Reply:
Check 👍🏻